package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "statsd-client",
-  "version": "0.4.5",
+  "version": "0.4.6",
```

Comment:
Please leave out the version bump, that'll be done as part of the release.

Reply:
cleanned.